libqtile/window.py

```diff
@@ -936,7 +963,7 @@ def cmd_static(self, screen, x=None, y=None, width=None, height=None):
         hook.fire("client_managed", s)
 
     def tweak_float(self, x=None, y=None, dx=0, dy=0,
-                    w=None, h=None, dw=0, dh=0):
+                    w=None, h=None, dw=0, dh=0, z=None, dz=0):
```

Comment:
Let's stay consistent and put `z=None` just after `y=None`, and same for `dz`. A quick grep in the codebase told me that we always call the function with the keywords.

Reply:
That sounds good. I didn't want to break anything.